tests/namespace/test_errors_namespace.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from pyvista.core.errors import PyVistaDeprecationWarning
+
+
+@pytest.mark.parametrize(
+    'name',
+    [
+        'AmbiguousDataError',
+        'DeprecationError',
+        'MissingDataError',
+        'NotAllTrianglesError',
+        'PointSetCellOperationError',
+        'PointSetDimensionReductionError',
+        'PointSetNotSupported',
+        'PyVistaDeprecationWarning',
+        'PyVistaEfficiencyWarning',
+        'PyVistaFutureWarning',
+        'PyVistaPipelineError',
+        'VTKVersionError',
+        'InvalidCameraError',
+        'RenderWindowUnavailable',
+    ],
+)
+def test_utilities_namespace(name):
+    with pytest.warns(PyVistaDeprecationWarning):
```

Comment:
Would it be overkill to separate the core errors from the plotting ones, and also asserting the correct warning message?

Reply:
Having thought about it a bit, we could add the new import path to the parameterisation, and also check that this path is what gets imported by the deprecated path. So I think this would be worth the conplexity.

Reply:
Done in 09611295faa4fbb6def1554958274ae979cc9411